userProfile/signals.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from .models import Profile
+
+
+@receiver(post_save, sender=User)
+def create_profile(sender, instance, created, **kwargs):
```

Comment:
Why does this need to be in a different file than the model? Something like [this](https://github.com/redhat-beyond/falcon/blob/main/users/models.py)

Reply:
In Django's documentation they recommended adding signals in this way.
It's supposed to prevent import problems.
![SmartSelect_20220328-160347_Chrome.jpg](https://user-images.githubusercontent.com/96747472/160403814-8f5ae808-2fe6-4c25-b422-6d812e4d1a34.jpg)
https://docs.djangoproject.com/en/4.0/topics/signals/